pkg/executor/read_write_splitting.go

```diff
@@ -29,7 +30,13 @@ import (
 	"github.com/cectc/dbpack/pkg/log"
 	"github.com/cectc/dbpack/pkg/mysql"
 	"github.com/cectc/dbpack/pkg/proto"
+	"github.com/cectc/dbpack/pkg/resource"
 	"github.com/cectc/dbpack/third_party/parser/ast"
+	"github.com/cectc/dbpack/third_party/parser/model"
+)
+
+const (
+	hintUseDB = "UseDB"
```

Comment:
Is there any hintType we can use?

Reply:
hint defined by https://github.com/CECTC/dbpack/blob/dev/third_party/parser/hintparser.y